src/pages/BirthDay.tsx

```diff
@@ -0,0 +1,86 @@
+import dayjs from "dayjs"
+
+import Layout from "../components/Layout/Layout"
+import { Button } from "../components/ui/button"
+import BirthDayList from "../components/BirthDay/BirthDayList"
+import { useState, useEffect } from "react"
+import { McyBirthdayApi } from "../api/mcyBirthdayApi"
+import { BirthdayInfo } from "../types/BirthdayInfo"
+
+const BirthDay = () => {
+  const [monthChipId, setMonthChipId] = useState<number>(dayjs().month() + 1)
+  const [birthDayData, setBirthDayData] = useState<BirthdayInfo[]>([]) // 파이어베이스 저장된 전체 생일 데이터
+  const [selectedData, setSelectedData] = useState<BirthdayInfo[]>([]) // 선택된 해당 월 생일 데이터
+
+  // 파이어베이스에서 데이터 가져오기
+  const fetchData = async (): Promise<void> => {
+    try {
+      const data = await McyBirthdayApi()
+      setBirthDayData(data)
+    } catch (error) {
+      console.error("Error fetching data: ", error)
+    }
+  }
+
+  // 해당 월 생일 데이터 필터링
+  const filterData = (): void => {
+    const filteredData = birthDayData.filter(item => {
+      const date = dayjs(item.date, "MM-DD")
+      const month = date.month() + 1
+      return month === monthChipId
+    })
+    setSelectedData(filteredData)
+  }
+
+  const handleChipClick = (targetId: number): void => {
+    setMonthChipId(targetId)
+  }
+
+  // 첫 렌더링시 파이어베이스에서 데이터 가져오기
+  useEffect(() => {
+    fetchData()
+  }, [])
+
+  // birthDayData가 변경되거나 monthChipId가 변경될 때 필터링 실행
+  useEffect(() => {
+    if (birthDayData.length > 0) {
+      filterData()
+    }
+  }, [birthDayData, monthChipId])
+
+  return (
+    <Layout>
+      <div className="w-full h-full flex flex-col">
+        {/* 월 선택 버튼 */}
+        <div className="w-[90%] h-[30%] mx-auto flex flex-col justify-center gap-[20px]">
```

Comment:
```suggestion
        <div className="w-[90%] h-[30%] mx-auto flex flex-col justify-center gap-[20px]">
```
높이는 굳이 퍼센트로 지정하기보단 고정값을 주는게 좋을듯..!

Reply:
모바일 기기마다 보이는 렌더링 영역이 어색해서 퍼센트로 줬습니다!!

Reply:
넵 확인했습니다!!